<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirDVD</class>
 <widget class="QDialog" name="IncluirDVD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir DVD</string>
  </property>
  <widget class="QDialogButtonBox" name="botao_ok_dvd">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>160</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="tit_incluirdvd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO DVD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_nomedvd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_precodvd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Preço:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_duracaodvd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recebe_nome_dvd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recebe_preco_dvd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recebe_duracao_dvd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botao_ok_dvd</sender>
   <signal>accepted()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_ok_dvd</sender>
   <signal>rejected()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
